packages/expo/package.json

```diff
@@ -54,14 +54,21 @@
     "@types/react": "*",
     "@types/react-dom": "*",
     "expo-auth-session": "^5.4.0",
+    "expo-local-authentication": "^14.0.1",
+    "expo-secure-store": "^13.0.1",
     "expo-web-browser": "^12.8.2",
+    "react": "18.2.0",
+    "react-native": "^0.74.2",
     "typescript": "*"
   },
   "peerDependencies": {
     "expo-auth-session": ">=4",
+    "expo-local-authentication": ">=14",
+    "expo-secure-store": ">=12",
```

Comment:
We probably need to add the above dependencies to [peerDependenciesMeta](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta) and mark them as optional, so we don't require users to install these if they don't need them